### PR TITLE
docs: Improve documentation for resource-service

### PR DESCRIPTION
**Pull request by thisthat**
Signed-off-by: Giovanni Liva <giovanni.liva@dynatrace.com>

Part of #7660
Part of #6996

### CODE REVIEW

File: resource-service/README.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
-# New Configuration Service
+# Resource Service :: The New Configuration Service
 
-The *resource-service* is a Keptn core component and used to manage resources for Keptn project-related entities, 
-i.e., project, stage, and service. The entity model is shown below. To store the resources with version control, a git 
-repository is used that is mounted as emptyDir volume.  Besides, this service has functionality to upload the git repository 
+The *resource-service* is a Keptn core component used to manage resources for Keptn project-related entities,
+i.e., project, stage, and service. The entity model is shown below. To store the resources with version control, a git
```
Comment (johannes-b) [inline resource-service/README.md:4]: Minor thing, but could you please change `git` to `Git` (search-replace in the entire document)

File: resource-service/README.md [hunk 2]
```diff
@@ -15,31 +19,45 @@ to any Git-based service such as GitLab, GitHub, Bitbucket, etc.
...
+To delete a deployed *resource-service*, use the file `deploy/service.yaml` from this repository
 and delete the Kubernetes resources:
 
 ```console
 kubectl delete -f deploy/service.yaml
 ```
+
+## Migration from the configuration-service
+
+Before migrating from the *configuration-service* to the *resource-service* it is recommended to (i) attach an upstream to your Keptn projects and (ii) do a [backup](https://keptn.sh/docs/0.15.x/operate/backup_and_restore/#back-up-configuration-service). If you set an upstream for all your Keptn projects, no additional steps are required.
```
Comment (johannes-b) [inline resource-service/README.md:56]: ... it is recommended to ...

shouldn`t it be 

.. it is required to ... ?

File: resource-service/README.md [hunk 3]
```diff
@@ -15,31 +19,45 @@ to any Git-based service such as GitLab, GitHub, Bitbucket, etc.
...
+
+Suppose you need the additional features provided by the *resource-service*,  such as HTTPS/SSH or Proxy, to configure your Keptn project with an upstream. In that case,
```
Comment (johannes-b) [inline resource-service/README.md:58]: Can you please change this paragraph to a listing:
* step 1
* step 2
* ...

File: resource-service/README.md (hunk 2 above)
Comment (thisthat) [reply resource-service/README.md:56]: Both setting the upstream and doing the backup are not strictly mandatory

File: resource-service/README.md (hunk 1 above)
Comment (StackScribe) [inline resource-service/README.md]: Do you mean i.e. here or e.g.?  In other words, are these the specific entities for which can manage resources (which is what i.e. means) or that these are examples of entities?

Comment (thisthat) [reply resource-service/README.md]: These are all specific entities that the resource-service handles and not examples

File: resource-service/README.md [hunk 4]
```diff
@@ -15,31 +19,49 @@ to any Git-based service such as GitLab, GitHub, Bitbucket, etc.
...
+   ------------           ------------           ------------
 ```
 
 ## Installation
 
-The *resource-service* is installed as a part of [keptn](https://keptn.sh)
+The *resource-service* replaces the *configuration-service*, hence only one of the two can be run at the same time.
+The *resource-service* can be enabled during the installation of Keptn setting the Helm value `control-plane.resourceService.enabled` to `true`.
+This flag changes the *configuration-service* `Service` to point towards the *resource-service* `Pod`.
+When the resource-service is enabled by default, the Kubernetes `Service` will change name to `resource-service`.
```
Comment (StackScribe) [inline resource-service/README.md:34]:
```suggestion
When the resource-service is enabled by default, the Kubernetes `Service` name changes to `resource-service`.

```

File: resource-service/README.md [hunk 5]
```diff
@@ -15,31 +19,49 @@ to any Git-based service such as GitLab, GitHub, Bitbucket, etc.
...
 ```
+
+## Migration from the configuration-service
+
+Before migrating from the *configuration-service* to the *resource-service* it is recommended to (i) attach an upstream to your Keptn projects and (ii) do a [backup](https://keptn.sh/docs/0.15.x/operate/backup_and_restore/#back-up-configuration-service). If you set an upstream for all your Keptn projects, no additional steps are required.
+
+Suppose you need the additional features provided by the *resource-service*,  such as HTTPS/SSH or Proxy, to configure your Keptn project with an upstream. In that case,
+you can also deploy the *resource-service* and configure the Git repositories later. For this, a backup is necessary.
+
+1. Do a backup of the [configuration-service](https://keptn.sh/docs/0.15.x/operate/backup_and_restore/#back-up-configuration-service).
```
Comment (StackScribe) [inline resource-service/README.md:61]:
```suggestion
1. Back up the [configuration-service](https://keptn.sh/docs/0.15.x/operate/backup_and_restore/#back-up-configuration-service).

```

**Review by StackScribe — DISMISSED** [review]
lgtm.  A.couple tiny suggestions for your consideration

**Comment by thisthat** [comment]
Thank you @StackScribe for your suggestions! I've updated the PR accordingly 😄 could you please have another look?

**Review by bacherfl — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]

**Pull request merged**